### ISSUE TITLE
Can't pair

### ISSUE THREAD

**Issue by headshot2017** [timawesomeness/PairWiimote#11]
After pressing the SYNC button on the wiimote and the LEDs start flashing, I tap "Nintendo RVL-CNT-01". It brings up the PIN input dialog for a second then closes right after, with the LEDs no longer blinking, and an error saying that the pairing failed.
A similar issue also happens when trying to pair through the device's Bluetooth settings page.

Samsung Galaxy A52, Android 12

![Screenshot_20220414-171522_Pair Wiimote](https://user-images.githubusercontent.com/36455681/163476943-8239a212-0260-49fe-8f9d-29b9d400cbda.png)

**Comment by kokbira** [comment]
The same for me, but I could connect after a lot of tries. As the app says, you must hold 1+2. I did that, keeping both pressed until I finished tapping 0000 on password and it worked. But all 4 leds is blinking and none of the buttons are working...

**Comment by ff66theone** [comment]
Pressing 1+2 instead of Sync is effectively not the same, as for some reason the PIN is different depending on which way you did. Personnally it didn't work with 1+2 but it did with Sync, though no initialization was done after...

**Comment by ff66theone** [comment]
Yeah after a bit of tests I can affirm that this issue only happens on Samsung devices with their stock ROMs
I have a rooted A5 2017 so that's why I finally got it to work correctly

**Comment by Kaddach2005** [comment]
<img width="1080" height="2312" alt="Image" src="https://github.com/user-attachments/assets/6f67e2b2-4656-4eea-90dd-b8e26ad9b3a0" />

<img width="1080" height="2312" alt="Image" src="https://github.com/user-attachments/assets/c882ec31-30ca-4fbf-bf94-7593173c324f" />
